Resfeber/CoreData/TripsController.swift

```diff
@@ -79,7 +97,7 @@ extension TripService {
         event.trip = trip
         
         do {
-            try CoreDataStack.shared.saveContext(context: context)
+            try coreDataStack.saveContext(context: context)
```

Comment:
These changes are all unnecessary too. I would change them back to the `CoreDataStack.shared`

Reply:
This is set up so the unit tests can create a mock CoreDataStack class and inject it into the TripsController class.